packages/client/src/components/Header/Header.tsx

```diff
@@ -60,7 +63,7 @@ export const Header = () => {
         className={s.nav}
       />
       <Dropdown menu={{ items: dropDownItems }}>
-        <div>nagibator322</div>
+        <div>{user&& user.login}</div>
```

Comment:
Тут можно не рисовать весь `div`, если нет `user`

```suggestion
        {!!user && (<div>{user.login}</div>)}
```

Reply:
> Тут можно не рисовать весь `div`, если нет `user`

Попробовал. Выдает ошибку - React.Children.only expected to receive a single React element child.

Нужен хотя бы один элемент.

Reply:
Тогда получается можно не рисовать весь родительский компонент

Reply:
> Тогда получается можно не рисовать весь родительский компонент

Ок. Сделал